src/App.tsx

```diff
@@ -1,14 +1,46 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[] | null>(null);
```

Comment:
```suggestion
  const [todos, setTodos] = useState<Todo[]>([]);
```
For array values you can set empty array as default value
It's easier to handle it properly in any case and allows to get rid of some redundant checks